packages/components/organisms/f-takeawaypay-activation/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.8.0
+------------------------------
+*August 13, 2021*
+
+### Changed
+- Cleanup styles
+
```

Comment:
### Added
- Loading spinner

Reply:
@DamianMullins could you please take a look once again? as you +1 the same comment